src/shared/components/MultiThemeSelectDropdown/MultiThemeSelectOptionGroupHeading.tsx

```diff
@@ -0,0 +1,38 @@
+import { Icon, IconName } from '@viaa/avo2-components';
+import React, { FC, useEffect, useState } from 'react';
+import { components, GroupHeadingProps } from 'react-select';
+
+const MultiThemeSelectOptionGroupHeading: FC<GroupHeadingProps> = (props) => {
+	const { GroupHeading } = components;
+	const [isOpen, setIsOpen] = useState<boolean>(false);
+	const [element, setElement] = useState<Element | null>();
+	const handleHeaderClick = () => {
+		if (!isOpen) {
+			element?.classList?.remove('collapsed');
+		} else {
+			element?.classList?.add('collapsed');
```

Comment:
better to use the BEM naming convention and use the --collapsed modifier, so we are sure we're not applying styles to any other elements in the site
```suggestion
			element?.classList?.add('c-multi-theme-select__group-heading--collapsed');
```